### DISCUSSION TITLE
How to validate if reg.body ?

### DISCUSSION THREAD

**Discussion by adamtretera** [prisma/prisma#12199] [Q&A] [answered]
How to validate if req.body ? is empty ?  without any 3rd party libary

**Comment by nurul3101** [comment, marked as answer]
Hey @adamtretera 👋 

Without using any external 3rd party libraries, you could check the length of keys of your request object. If it is zero then it would mean your request is empty.

This condition should be helpful to you
```javascript
if (Object.keys(req.body).length === 0) {
   // Body is empty
}
```

**Comment by ludralph** [comment]
Hi there,

To keep our discussions organized and focused on the most relevant topics, we’re reviewing and tidying up our backlog. As part of this process, we’re closing discussions that have already been marked as answered but remain open.

If this discussion still requires further input or clarification, feel free to reopen it or start a new one with updated details. Your contributions are invaluable to the community, and we’re here to help!

For more details about our priorities and vision for the future of Prisma ORM, check out our latest blog post: https://www.prisma.io/blog/prisma-orm-manifesto.

Thank you for your understanding and ongoing support of the Prisma community!
